roles/openstack_version/tasks/main.yml

```diff
@@ -18,6 +18,18 @@
   set_fact:
     rhosp_version: "{{ version.split()[5] }}"
 
+- name: set rhosp major version
+  set_fact:
+    rhosp_major: "{{ version | regex_replace('^Red Hat OpenStack Platform release ([0-9]+)\\.\\d+.*\n', '\\1') }}"
```

Comment:
can you please add stockpile_ prefix to the name ?

Reply:
Done